packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -654,6 +654,24 @@ export class DataStores implements IDisposable {
         }
     }
 
+    public revive(usedRoutes: string[]) {
```

Comment:
You can do this as part of `updateUsedRoutes`. It already does most of this work and you just need to hook into that to untombstone data stores.